CSharpMath.CoreTests/TypesetterTests.cs

```diff
@@ -33,15 +33,15 @@ public class TypesetterTests {
     void TestOuter(string latex, int rangeMax, double ascent, double descent, double width,
         params System.Action<IDisplay<TFont, TGlyph>>[] inspectors) =>
       TestList(rangeMax, ascent, descent, width, 0, 0, LinePosition.Regular, Range.UndefinedInt, inspectors)
-      (Typesetter.CreateLine(LaTeXParserTest.ParseLaTeX(latex), _font, _context, LineStyle.Display));
+      (ParseLaTeXToDisplay(latex));
 
-    [Theory, InlineData("x"), InlineData("2")]
-    public void TestSimpleVariable(string latex) =>
+    [Theory, InlineData("x", "𝑥"), InlineData("2", "2"), InlineData(".", "."), InlineData("𝑥", "𝑥")]
+    public void TestSingleCharacter(string latex, string text) =>
```

Comment:
Is it known what this test is for? If not, we should delete. If so, we should rename/document.

Reply:
Makes sure that a single character of type Variable/Number/Punctuation/Ordinary(surrogates) measures as the same one-character size.

Reply:
I copied this comment in but don't understand the terms involved. You'll have to judge whether this is resolved.

Reply:
Clarified